.github/workflows/stlab.yml

```diff
@@ -30,6 +30,14 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - uses: mymindstorm/setup-emsdk@v11
+        if: ${{ startsWith(matrix.config.compiler, 'emscripten') }}
```

Comment:
Neither apt nor apt-get have an emscripten or emsdk package, but I've removed this dependency in favor of a manual installation of emsdk.

Notably, it requires we modify bash_profile, so you'll see all subsequent emscripten steps will use `shell: bash -l {0}`, in order to override GitHub Actions' default of a --noprofile shell. See [here](https://github.com/orgs/community/discussions/25061#discussioncomment-3246364) for more.

Reply:
I have it installed on my Ubuntu machine. See: https://packages.ubuntu.com/jammy/emscripten

Reply:
Those are considered [unofficial packages](https://emscripten.org/docs/getting_started/downloads.html#installation-using-unofficial-packages) by the Emscripten team, and are not officially supported. I've opted to stick strictly to their installation guidelines and clone/install their repo instead of adding the aforementioned dependency.